src/envo/envo-edit.owl

```diff
@@ -41989,8 +42006,147 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501354> <http://purl.obolibrar
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501367> <https://orcid.org/0000-0002-2738-824X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501367> "2021-12-09T00:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501367> "pneumatic conveyor"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501367> <http://purl.obolibrary.org/obo/ENVO_03501355>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501367> <http://purl.obolibrary.org/obo/ENVO_03501356>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501368> (conveyor roller)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wiktionary.org/wiki/roller>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501368> "A manufactured product which is a cylindrical (or approximately cylindrical) item used under a heavy object to facilitate moving it."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501368> <https://orcid.org/0000-0002-2738-824X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501368> "2021-12-09T00:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03501368> "Usually several conveyor rollers are organized in a linear fashion to convey an object between locations."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501368> "conveyor roller"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501368> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501369> (seam surface)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://vocab.getty.edu/page/aat/300228472>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501369> "A two-dimensional spatial region where one material entity touches or is attached to itself or another material entity along some length which is exposed to an environment."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501369> <https://orcid.org/0000-0002-2738-824X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501369> "2021-12-09T00:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501369> "seam surface"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501369> <http://purl.obolibrary.org/obo/BFO_0000006>)
```

Comment:
don't make direct subclasses to BFO - and this has no axioms connecting it to ENVO at all!

this is quite problematic in general. The definition you have provided fits anatomical junctions in uberon. So if ENVO accepts this then uberon should really subclass this to avoid gaps in OBO.

But I really think something this generic doesn't belong in ENVO

however, judging by the context you provide in the issue, I don't think you want something this general. I think you want something like "machinery seam surface". Even with this I have many questions, but if things are at least well scoped when they come in then we can iterate.